ios/objective-c/AmpliObjectiveCSampleApp/README.md

```diff
@@ -0,0 +1,34 @@
+# Ampli iOS SDK (Objective-C)
```

Comment:
@qingzhuozhen can you please also add links to the ObjC and Swift examples from the top-level `ampli-examples/README.md`

Reply:
Sure, let me update it

Reply:
updated